src/codegen/swagger.yaml

```diff
@@ -0,0 +1,371 @@
+---
+swagger: "2.0"
+info:
+  description: "This is a sample server role-service."
+  version: "1.0.0"
+  title: "Swagger role-service"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "wit2022.v1@gmail.com>"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "medicalcenter.roleservice"
```

Comment:
I think we need to have the actual hostname that we set using docker compose.

Reply:
What do you mean?